docs/kubernetes/how-to-deploy-jenkins-x-in-linode-kubernetes-engine/index.md

```diff
@@ -0,0 +1,365 @@
+---
+author:
+  name: Daniele Polencic
+  email: daniele@learnk8s.io
+description: 'Learn how to deploy Jenkins X in Linode Kubernetes Engine.'
+og_description: 'Learn how to deploy Jenkins X in Linode Kubernetes Engine.'
+keywords: ['kubernetes','pipelines','ci/cd','kubernetes','jenkins x','jenkins']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-07-11
+modified_by:
+  name: Linode
+title: "How to deploy Jenkins X in Linode Kubernetes Engine"
+h1_title: "How to deploy Jenkins X in Linode Kubernetes Engine"
+contributor:
+  name: Daniele Polencic
+  link: https://github.com/danielepolencic
+external_resources:
+- '[How to run Jenkins X Boot](https://jenkins-x.io/docs/install-setup/boot/how-it-works/)'
+- '[Creating and importing projects in Jenkins X](https://jenkins-x.io/docs/create-project/creating/)'
+---
+
+## What is Jenkins X?
+
+[Jenkins X](https://jenkins-x.io) is CI/CD tool that automates the management of the environments and the promotion of new versions of applications between environments.
+
+## Before You Begin
+
+1. You will need an account on GitHub. If you don't have one, [you can register here](https://github.com/join).
+1. You will need an account on DockerHub. If you don't have one, [you can register here](https://hub.docker.com/signup).
+
+## Create an LKE Cluster
+
+Follow the instructions in [Deploying and Managing a Cluster with Linode Kubernetes Engine Tutorial](https://www.linode.com/docs/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/) to create and connect to an LKE cluster.
+
+We recommend using 3 Linode 16GB of memory to start with.
+
+![Creating a Kubernetes cluster on Linode](create-cluster.png)
+
+You can verify that the installation is successful with:
+
+```bash
+kubectl get nodes
+```
+
+The output should similar to:
+
+{{< output >}}
+NAME                        STATUS   ROLES    AGE   VERSION
+lke7189-9006-5f05145fc9a3   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f051460a1e2   Ready    <none>   8h    v1.17.3
+lke7189-9006-5f0514617a87   Ready    <none>   8h    v1.17.3
+{{</ output>}}
+
+## Install `jx`
+
+To install Jenkins X, you need the command-line tool `jx`.
+
+You can find the instruction on [how to install `jx` on the official documentation.](https://jenkins-x.io/docs/install-setup/install-binary/)
+
+You can verify that the installation is successful with:
+
+```bash
+jx version
+```
+
+The output should be similar to:
+
+{{< output >}}
+Version        2.1.80
+Commit         b2bd447
+Build date     2020-06-23T15:34:17Z
+Go version     1.13.8
+Git tree state clean
+{{</ output>}}
+
+## Bootstrap Jenkins X
+
+Create a new folder and execute the `jx boot` command to start a new project:
```

Comment:
We need to execute the following commands in the same terminal that you connected to the LKE cluster, else and error appears:
<img width="566" alt="image" src="https://user-images.githubusercontent.com/37059749/88457231-6869a800-cea2-11ea-8024-98249737288f.png">

Reply:
This is true for all Kubernetes tutorials that use `export KUBECONFIG=XXX`.

If you change the terminal, the `export` statement isn't valid.

We have a couple of options here:
- The instruction should be included in the original tutorial here: https://www.linode.com/docs/kubernetes/deploy-and-manage-a-cluster-with-linode-kubernetes-engine-a-tutorial/
- We include the message here (and perhaps retrofit the change to the rest of the Kubernetes tutorials?)

Reply:
Yes!!! may be we could include it here and retrofit in all related tutorials.
@andystevensname  may be this can be done during the copy edit?

Reply:
I'll make the note in the guide, and a task for updating our other guides.